lib/utils.pm

```diff
@@ -338,11 +340,11 @@ C<$check_typed_password> will default to C<0>.
 sub unlock_if_encrypted {
     my (%args) = @_;
     $args{check_typed_password} //= 0;
+    my $password = check_var('SYSTEM_ROLE', 'Common_Criteria') ? $security::config::strong_password : $testapi::password;
```

Comment:
This fits better in: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/2a0c39849ca58afff6af72ddfb05b886467166e5/lib/main_common.pm#L149

Reply:
See also the following message, about documenting the test data https://suse.slack.com/archives/C02CANHLANP/p1728634460708519?thread_ts=1728632342.244079&cid=C02CANHLANP

Reply:
thanks, I'll do that!

Reply:
actually, that password is just for disk encryption and not for the user itself.